docs/configuration.rst

```diff
@@ -410,6 +410,12 @@ RESPONSE_VALIDATION
     that they conform to the protocol. This should only be used for development
     purposes.
 
+LANDING_MESSAGE_HTML
+    The server provides a simple landing page at its root. By setting this 
+    value to point at an HTML snippet it is possible to customize the landing 
```

Comment:
This should be defined as an html `block element' (paragraph,
div, etc), not a snippet, which has no formal definition.  The
resulting page should be valid HTML and if the template is
inserted in a place where a block is valid and then block is
valid, it should be valid.
 

Maciek Smuga-Otto <notifications@github.com> writes:
> In docs/configuration.rst:
> 
> > @@ -410,6 +410,12 @@ RESPONSE_VALIDATION
> >      that they conform to the protocol. This should only be used for development
> >      purposes.
> >
> > +LANDING_MESSAGE_HTML
> > +    The server provides a simple landing page at its root. By setting this
> > +    value to point at an HTML snippet it is possible to customize the landing
> 
> at a file containing an HTML snippet
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub*
>